widgetssdk/src/main/java/com/glia/widgets/webbrowser/WebBrowserActivity.kt

```diff
@@ -7,7 +7,14 @@ import android.os.Bundle
 import com.glia.widgets.base.FadeTransitionActivity
 import com.glia.widgets.databinding.WebBrowserAvtivityBinding
 
-
+/**
+ * Glia internal class.
+ *
+ * Will be automatically added to integrator Manifest by Manifest merger during compilation.
+ *
+ * This activity is used to display simple web pages. For example, used for displaying client's Terms and Conditions or
```

Comment:
Do you mean that when engagement confirmation dialog offers me to read Terms and Conditions, there's a link to the terms and conditions page and it is shown within the app?

Reply:
@kadriiiii yes, it's part of the new functionality added together with live observation. Here's the article for more context: https://docs.glia.com/glia-mobile/docs/ios-user-consent-and-disclaimers